resources/scripts/components/server/Console.tsx

```diff
@@ -201,8 +201,12 @@ export default () => {
         };
     }, [ connected, instance ]);
 
+    function openConsole () {
```

Comment:
Use const instead of a function to better match the overall code-style of this file.

Reply:
Fixed in https://github.com/pterodactyl/panel/pull/3904/commits/b1ed950cff39e1bbe3ed53e697fe34a9b91e7eb3